set-mongo/src/main/java/eu/europeana/set/web/model/WebUserSetImpl.java

```diff
@@ -22,7 +22,7 @@
 
 @JsonPropertyOrder({ WebUserSetModelFields.ID, WebUserSetModelFields.TYPE, WebUserSetModelFields.TITLE, WebUserSetFields.DESCRIPTION,
         WebUserSetModelFields.VISIBILITY, WebUserSetModelFields.IS_DEFINED_BY, WebUserSetModelFields.ITEMS, WebUserSetModelFields.CREATOR,
-        WebUserSetModelFields.CREATED, WebUserSetModelFields.MODIFIED, WebUserSetModelFields.TOTAL, WebUserSetFields.NEXT,
+        WebUserSetModelFields.CREATED, WebUserSetModelFields.MODIFIED,WebUserSetModelFields.SUBJECT, WebUserSetModelFields.TOTAL, WebUserSetFields.NEXT,
```

Comment:
We have to ask Hugo for the ordering, but I assume that the next field should be placed before or after description

Reply:
have placed before description